index.html

```diff
@@ -9256,7 +9264,20 @@ <h5>Presentational Role Inheritance</h5>
 			<rdef>textbox</rdef>
 			<div class="role-description">
 				<p>A type of input that allows free-form text as its value.</p>
+				<p>Authors MUST limit its children to completely presentational elements (like a spacer image, decorative graphic, or clearing element).</p>
```

Comment:
Looks like an unrelated change snuck into this PR! Can you delete this line.

Reply:
Thanks @spectranaut I removed the unrelated change in commit [81cd343](https://github.com/chlane/aria/commit/81cd343772f02835c1d27cbf45553726508ee064)